File: Java/leetcode75/src/test/java/org/phoenix/leetcode/challenges/Problem44_NumberOfProvincesTest.java
package org.phoenix.leetcode.challenges;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class Problem44_NumberOfProvincesTest {

    private final Problem44_NumberOfProvinces test = new Problem44_NumberOfProvinces();

    @Test
    void findCircleNum() {
        int[][] isConnected = new int[][]{{1, 1, 0}, {1, 1, 0}, {0, 0, 1}};

        assertEquals(2, test.findCircleNum(isConnected));
    }
}

File: JavaScript/leetcode75/src/test/Problem44_NumberOfProvinces.test.js
const findCircleNum = require("../main/Problem44_NumberOfProvinces");

test("should return total number of provinces", () => {
  const isConnected = [
    [1, 1, 0],
    [1, 1, 0],
    [0, 0, 1],
  ];

  expect(findCircleNum(isConnected)).toBe(2);
});
